### ISSUE TITLE
Migrations is not working

### ISSUE THREAD

**Issue by soaresdiogo** [innovativeinstitution/MicroRabbit#1]
Hi,
I did the download and the migrations is not working is ocurring this error:
`Could not load assembly 'MicroRabbit.Banking.Data'. Ensure it is referenced by the startup project 'MicroRabbit.Domain.Core'.`

**Comment by soaresdiogo** [comment]
I tried with VsCode and VS2019

**Comment by soaresdiogo** [comment]
It still isn't working:
Could not load assembly 'MicroRabbit.Banking.Data'. Ensure it is referenced by the startup project 'MicroRabbit.Domain.Core'.

And the line there isn't in Startup.cs, I saw in your project
services.AddTransient<IEventBus, RabbitMQBus>();

**Comment by setki** [comment]
Yes, also issues over here. I tried the step-by-step manual and got stuck on the "Add-Migration" part.
"Your startup project .... doesn't reference Microsoft.EntityFrameworkCore.Design."
It wasn't told so in the video. Not sure what to do.
